### ISSUE TITLE
`isAccessibilityElement` should not return `false` for `NSUIView`

### ISSUE THREAD

**Issue by beckysippert** [danielgindi/Charts#4253]
Since `isAccessibilityElement` is a final, overridden property of `NSUIView` in `Platform+Accessibility.swift`, it is essentially read-only, since the setter has no effect. The comment explains that it is to make children accessible, but this is something that should be configurable since `UIView` allows it. The use case I am running into is that the chart data is repeated in another format within the same view, rendering the chart data redundant to a voiceover listener. The chart is purely a visual aid in this case.
